### PR TITLE
Remove CoordinatorBuilder and refactor CoordinatorApp tests

### PR DESCRIPTION
**Pull request by cfournie**
This PR:
  - Removes the `CoordinatorBuilder` in favour of `CoordinatorApp.xml` being used to construct coordinators; and
  - Refactors the `CoordinatorApp` tests to cover the same options that the `CoordinatorBuilder` covered.

Fixes https://github.com/Shopify/pyoozie/issues/34.

### CODE REVIEW

File: tests/pyoozie/test_tags.py [hunk 1]
```diff
@@ -387,54 +380,63 @@ def parse_datetime(string):
...
 
-def test_coordinator(coordinator_xml, expected_coordinator_options):
-    actual = tags.CoordinatorApp(**expected_coordinator_options).xml()
-    assert tests.utils.xml_to_dict_unordered(coordinator_xml) == tests.utils.xml_to_dict_unordered(actual)
+def test_minimal_coordinator(minimal_coordinator_options):
+    actual_xml = tags.CoordinatorApp(**minimal_coordinator_options).xml(indent=True)
+    actual_dict = tests.utils.xml_to_dict_unordered(actual_xml)
 
+    expected_xml = None
+    with open('tests/data/minimal_coordinator.xml', 'r') as fh:
```
Comment (kmtaylor-github) [inline tests/pyoozie/test_tags.py]: Not a fixture?

Comment (cfournie) [inline tests/pyoozie/test_tags.py]: Meh, no other tests use it.

File: tests/pyoozie/test_tags.py [hunk 2]
```diff
@@ -387,54 +380,63 @@ def parse_datetime(string):
...
-    assert tests.utils.xml_to_dict_unordered(coordinator_xml) == tests.utils.xml_to_dict_unordered(actual)
+    actual_xml = tags.CoordinatorApp(**minimal_coordinator_options).xml(indent=True)
+    actual_dict = tests.utils.xml_to_dict_unordered(actual_xml)
 
+    expected_xml = None
+    with open('tests/data/full_coordinator.xml', 'r') as fh:
+        expected_xml = fh.read()
+    expected_dict = tests.utils.xml_to_dict_unordered(expected_xml)
 
-def test_coordinator_with_controls_and_more(coordinator_xml_with_controls, expected_coordinator_options):
```
Comment (kmtaylor-github) [inline tests/pyoozie/test_tags.py]: Is the `coordinator_xml_with_controls` fixture still used, or can it be :fire:'d?

Comment (cfournie) [inline tests/pyoozie/test_tags.py]: Can be 🔥 't

**Pull request merged**